app/clConfig.ts

```diff
@@ -0,0 +1,3 @@
+export default interface CLConfig {
+    excludedServers: Set<string>;
```

Comment:
Did you test this? How can a JSON file contain a Set?

Reply:
sorry. I didn't test it. I pushed new commits that should fix it